### DISCUSSION TITLE
How to change scale of a model or position of a model ?

### DISCUSSION THREAD

**Discussion by rianlukog** [google/model-viewer#2524] [Q&A]
I have added a working code to my website for a model, I only have width and height parameters for the window, How can I change the Position or scale of the model?

Attaching code below

![GITHUG1](https://user-images.githubusercontent.com/85927224/123785891-a9f6a480-d8f6-11eb-996c-6c4ed31e39a9.png)

**Comment by elalish** [comment]
Try following this [example](https://modelviewer.dev/examples/scenegraph/#transforms). For position, we expose the camera-target attribute instead of moving the model itself.
